lib/nexmo_markdown_renderer.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 GEM_ROOT = File.expand_path("../..", __FILE__)
+DOCS_BASE_PATH = (defined?(Rails) && Rails.configuration.docs_base_path) ? Rails.configuration.docs_base_path : '.'
```

Comment:
I'd expect this to use the following options:

* `Rails.configuration.docs_base_path`
* `ENV['DOCS_BASE_PATH']`
* `.`

Reply:
👍 100% I'll fix that.